### ISSUE TITLE
Impossible to uprade jbang to latest

### ISSUE THREAD

**Issue by mbarbero** [jbangdev/homebrew-tap#4]
It seems that there is java version detection issue:

```
❯ brew upgrade
==> Upgrading 1 outdated package:
jbangdev/tap/jbang 0.53.1 -> 0.53.2
==> Upgrading jbangdev/tap/jbang 0.53.1 -> 0.53.2
jbang: Java 1.8+ is required for this software.
Install AdoptOpenJDK with Homebrew Cask:
  brew cask install adoptopenjdk
Error: jbang: An unsatisfied requirement failed this build.
```

While I have (multiple) adoptopenjdk casks already installed

```
❯ brew list --cask
adoptopenjdk                    adoptopenjdk11                  adoptopenjdk8                  
adoptopenjdk-openj9             adoptopenjdk11-openj9           adoptopenjdk8-openj9            
```

They are all properly listed by the `/usr/bin/java` system binary

```
❯ /usr/libexec/java_home -V
Matching Java Virtual Machines (6):
    15.0.1 (x86_64) "AdoptOpenJDK" - "AdoptOpenJDK (OpenJ9) 15" /Library/Java/JavaVirtualMachines/adoptopenjdk-15-openj9.jdk/Contents/Home
    15 (x86_64) "AdoptOpenJDK" - "AdoptOpenJDK 15" /Library/Java/JavaVirtualMachines/adoptopenjdk-15.jdk/Contents/Home
    11.0.9 (x86_64) "AdoptOpenJDK" - "AdoptOpenJDK (OpenJ9) 11" /Library/Java/JavaVirtualMachines/adoptopenjdk-11-openj9.jdk/Contents/Home
    11.0.9 (x86_64) "AdoptOpenJDK" - "AdoptOpenJDK 11" /Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home
    1.8.0_275 (x86_64) "AdoptOpenJDK" - "AdoptOpenJDK 8" /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home
    1.8.0_275 (x86_64) "AdoptOpenJDK" - "AdoptOpenJDK (OpenJ9) 8" /Library/Java/JavaVirtualMachines/adoptopenjdk-8-openj9.jdk/Contents/Home
/Library/Java/JavaVirtualMachines/adoptopenjdk-11-openj9.jdk/Contents/Home
```

and here is the report for

```
❯ java -version
openjdk version "11.0.9" 2020-10-20
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.9+11)
Eclipse OpenJ9 VM AdoptOpenJDK (build openj9-0.23.0, JRE 11 Mac OS X amd64-64-Bit Compressed References 20201023_788 (JIT enabled, AOT enabled)
OpenJ9   - 0394ef754
OMR      - 582366ae5
JCL      - 3b09cfd7e9 based on jdk-11.0.9+11)
```

I also tried to install the `openjdk` formula, but `brew upgrade` is still outputting the same issue.

(Note that I switched to Big Sur over the weekend, but I was already hitting the issue last week, while still on Catalina).

**Comment by mbarbero** [comment]
I just tried removing jbang

```
❯ brew remove jbang
Uninstalling /usr/local/Cellar/jbang/0.53.1... (59 files, 5.8MB)
```

and now I cannot install it anymore

```
❯ brew install jbangdev/tap/jbang
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 1 formula.

==> Installing jbang from jbangdev/tap
jbang: Java 1.8+ is required for this software.
Install AdoptOpenJDK with Homebrew Cask:
  brew cask install adoptopenjdk
Error: An unsatisfied requirement failed this build.
```

**Comment by maxandersen** [comment]
gah - seems i only removed the cask dep not java 1.8... i'll get that fixed in next build.

**Comment by maxandersen** [comment]
btw. i'm not seeing the error when installing so i'm not sure what is different between our setups.

**Comment by mbarbero** [comment]
> gah - seems i only removed the cask dep not java 1.8... i'll get that fixed in next build.

Thanks



> btw. i'm not seeing the error when installing so i'm not sure what is different between our setups.

Let me know if you want some additional infos on my setup to troubleshoot the issue.

**Comment by mbarbero** [comment]
It has been fixed a couple of releases ago (v0.54 IIRC). Thanks.
